test/e2e/tests/setup_e2e_test.go

```diff
@@ -35,6 +36,9 @@ import (
 var (
 	project         = flag.String("project", "", "Project to run tests in")
 	serviceAccount  = flag.String("service-account", "", "Service account to bring up instance with")
+	zones           = flag.String("zones", "us-central1-c,us-central1-b", "Zones to run tests in. If there are multiple zones, separate each by comma")
+	machine         = flag.String("machine", "n1-standard-1", "Type of machine to provision instance on")
+	imageURL        = flag.String("image-url", "projects/debian-cloud/global/images/family/debian-11", "OS image url to get image from")
```

Comment:
since we have moved to distroless, do we need to change to that here too? @pwschuurman